test/Spaces/unit_spaces.jl

```diff
@@ -25,6 +25,48 @@ import ClimaCore.DataLayouts: IJFH, VF, slab_index
 
 on_gpu = ClimaComms.device() isa ClimaComms.CUDADevice
 
+@testset "2D spaces with mask" begin
```

Comment:
It would be good to add small tests for `PointSpace` and an `Extruded` space too, to check the `get_mask` definitions in `src/Spaces/Spaces.jl`

Reply:
I agree that we can / should write tests for extruded spaces, but I don't know that masks make sense for point spaces. If you've called `Fields.column` on a field, it's already too late to apply a mask operation. Same for FiniteDifferenceSpace-- if you mask a FD space, then no operations will ever happen.

Reply:
I've added extruded spaces tests